client/my-sites/site-logs/components/site-logs-table/style.scss

```diff
@@ -41,3 +42,27 @@ $border-color: #d3dae6;
 .site-logs-table__skeleton-table-cell {
 	height: 24px;
 }
+
+tr.site-logs-table__table-row-expanded {
+	background: var(--studio-gray-0) !important;
+	td {
+		border-top: none;
+		vertical-align: top;
+	}
+}
+
+.site-logs-table__expanded-content {
+	padding-left: 30px;
+	font-size: 0.875rem;
+	tr {
+		border: none !important;
+	}
+}
+
+.site-logs-table__expanded-content-info {
+	word-break: break-word;
+	word-wrap: break-word;
+	max-width: 70%;
```

Comment:
Hmmm, tricky. the whole point of expanding the row is so that you don't have to use the horizontal scrollbar. So we have to have some sort of width specified. The downside of this particular width is that as the screen size gets smaller even the expanded content needs the horizontal scrollbar.

I found another method where, rather than adding a `max-width` on the `.site-logs-table__expanded-content-info` we add a width to the `.site-logs-table__expanded-content` element. It looks something like this:

```scss
.site-logs-table__expanded-content {
	// This makes the expanded content fit inside the viewport
	// The 440px takes into account the Calypso sidebar and the necessary table padding
	width: calc(100vw - 440px);
}

body.is-sidebar-collapsed .site-logs-table__expanded-content {
	// We'd need another rule for when the sidebar is collapsed
	width: calc(100vw - 200px);
}

// And then I guess other rules for mobile breakpoints?
```

Reply:
I've used this as a base for a solution for all viewport sizes. On mobile, I made it a single column, as there isn't enough space for two columns.

<img width="503" alt="Screenshot 2566-04-10 at 15 34 21" src="https://user-images.githubusercontent.com/6851384/230865250-e4080bf1-de6c-4cb2-87d9-dd407bfc4016.png">